### ISSUE TITLE
Friday lunch

### ISSUE THREAD

**Issue by ultimike** [mherchel/fldc-issues#150]
Need to organize and provide lunch for training attendees.

In 2018, we ordered 5 Publix sandwich platter for 80 folks - probably 1 or 2 short. Also - need more vegetarian options.

**Comment by ultimike** [comment]
@lscook Interested in handling Friday food (in addition to Sunday food)?

-mike

**Comment by lscook** [comment]
Yes. I will need help picking up the platters and everything that goes with it. Mike H. helped me last year. Is it possible to arrange it so that no one misses a part of their training in order to help me? That was an issue last year.

**Comment by lscook** [comment]
Looking for volunteers to help with going to pick up the food and serving the food. Another great opportunity to network. Everyone will recognize you if you give them food.

**Comment by andrewmriley** [comment]
I can help serve.

**Comment by capysara** [comment]
I can grab the food in the morning. Do we have access to the fridge there? Otherwise, I can bring a cooler.

**Comment by lscook** [comment]
That's a really good idea. You'll need help. I think you and I can handle it but we'll need help unloading and carrying it to the lunch room. It's a huge amount of food. I don't know if we have access to refrigerators but that would solve the problem of folks missing part of their training to go get the food. Is there someone we can ask?

**Comment by ultimike** [comment]
We do, but it's always hit-or-miss how much room is available...

**Comment by lscook** [comment]
I will have to give Publix a week notice so I will need an approximate training attendee count by Feb. 8, 2019.
List:
Variety of Sub platters: vegetable, cheese, ham, turkey, roast beef.
Variety of Chips
Hand sanitizer
Serving utensils
Disposable plates
Napkins
If you think of anything else please add it to the list.

**Comment by lscook** [comment]
We need a place to stash at least 7 platters if we have 80 people again.

**Comment by ultimike** [comment]
We won't have that. Best to pick up the platters around 11am so they don't
need to be refrigerated.

-mike

**Comment by lscook** [comment]
What's the latest attendee count for Friday lunch? Time to call Publix and give them a heads up. Documented for future FLDCs: Publix Super Market at Town Park, 12231 E Colonial Dr, Orlando, FL 32826 Phone # (407) 482-1244. Please double check the address for me.

**Comment by ultimike** [comment]
As of today, we have 76 folks signed up for Friday workshops. Counting instructors and last week signups, I think somewhere around 100 is a good number. Thoughts?

Also - yes, that is the proper Publix.

-mike

**Comment by lscook** [comment]
Called Publix Super Market at Town Park, 12231 E Colonial Dr, Orlando, FL 32826 Phone # (407) 482-1244 and spoke with Deli Manager Corey and gave him a heads up. Ordered 8 platters, 2 each of turkey, ham, italian, and veggie to be picked up at 11 am on Friday the 15th.  Estimated Total $503.92. If there are any changes to this order please contact me asap. Total number of platters is 3 more than we had last year. Each platter serves 30 - 36 people.

**Comment by ultimike** [comment]
Thank you!!!

**Comment by lscook** [comment]
Notes for next year: We need to add lysol wipes to the list for clean up and maybe some kind of signage so we won't have to keep repeating what's on the sandwiches. We ordered 3 more platters in 2019, (8 platters) than in 2018 and had one left over. The overall favorite was italian on wheat. Total cost ~ $650.00
New list:
Variety of Sub platters: vegetarian, ham, turkey, italian.
Variety of Chips (6 bags of 18)
Hand sanitizer (2)
Serving utensils (we can reuse the ones from this year for next year)
Disposable plates
Napkins
cups
drinks
lysol wipes
something to make signs with (paper plates and a marker and tape)
